app/scripts/controllers/swaps.test.js

```diff
@@ -377,7 +377,7 @@ describe('SwapsController', function () {
         assert.strictEqual(gasEstimate, bufferedGasLimit);
         assert.strictEqual(
           gasEstimateWithRefund,
-          `0x${new BigNumber(maxGas, 10)
+          `0x${new BigNumberjs(maxGas, 10)
```

Comment:
Could we just use the `ethers` version of BigNumber here too? And get rid of `bignumber.js` here?

Reply:
It will cause a test to fail because ethers BigNumber adds 0 when we covert to hex value. 
<img width="600" alt="image" src="https://user-images.githubusercontent.com/97883527/185597226-ebba40e1-0294-4692-895f-291fd2d992ae.png">